geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/classpath/ClassPathUtils.scala

```diff
@@ -98,6 +98,37 @@ object ClassPathUtils extends LazyLogging {
     files ++ children
   }
 
+  /**
+    * Finds URLs of files based on a system property
+    *
+    * @param prop
+    * @return
+    */
+  def getFilesFromSystemProperty(prop: String): Seq[File] = {
+    Option(System.getProperty(prop)) match {
+      case Some(path) => path.toString().split(":").map(new File(_)).filter{ file =>
+        val isDir = file.isDirectory
+        if (!isDir) logger.warn(s"${file} : Is not a directory")
+        isDir
+      }.toSeq.flatMap(loadFilesFromFolder)
+      case None =>
+        logger.debug(s"No files loaded onto classpath from system property: ${prop}")
+        Seq.empty
+    }
+  }
+
+  /**
+    * Recursively searches folders for all files
+    *
+    * @param dir
+    * @return
+    */
+  def loadFilesFromFolder(dir: File): Seq[File] = {
+    val files = Option(dir.listFiles()).toSeq.flatten
+    val children = Option(dir.listFiles(folderFileFilter)).toSeq.flatten.flatMap(loadFilesFromFolder)
+    files ++ children
```

Comment:
Does this lead to any duplication?  Does a sub-directory get listed twice ever?  Does that even matter;)?